CNF org.bouncycastle.operator.OperatorCreationException

This occurs at every restart of S1595, but I haven't seen it cause any issues.

This is one of the first log entries...

```
2019-05-18 20:22:12.401 [ERROR] [org.openhab.core.io.jetty.certificate] - FrameworkEvent ERROR - org.openhab.core.io.jetty.certificate
org.osgi.framework.BundleException: Error starting module.
        at org.eclipse.osgi.container.Module.doStart(Module.java:590) ~[?:?]
        at org.eclipse.osgi.container.Module.start(Module.java:449) ~[?:?]
        at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1634) ~[?:?]
        at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1614) ~[?:?]
        at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1585) ~[?:?]
        at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1528) ~[?:?]
        at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1) ~[?:?]
        at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230) [?:?]
        at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:340) [?:?]
Caused by: java.lang.NoClassDefFoundError: org/bouncycastle/operator/OperatorCreationException
        at java.lang.Class.getDeclaredConstructors0(Native Method) ~[?:?]
        at java.lang.Class.privateGetDeclaredConstructors(Class.java:2671) ~[?:?]
        at java.lang.Class.getConstructor0(Class.java:3075) ~[?:?]
        at java.lang.Class.newInstance(Class.java:412) ~[?:?]
        at org.eclipse.osgi.internal.framework.BundleContextImpl.loadBundleActivator(BundleContextImpl.java:763) ~[?:?]
        at org.eclipse.osgi.internal.framework.BundleContextImpl.start(BundleContextImpl.java:716) [?:?]
        at org.eclipse.osgi.internal.framework.EquinoxBundle.startWorker0(EquinoxBundle.java:933) ~[?:?]
        at org.eclipse.osgi.internal.framework.EquinoxBundle$EquinoxModule.startWorker(EquinoxBundle.java:309) ~[?:?]
        at org.eclipse.osgi.container.Module.doStart(Module.java:581) ~[?:?]
        ... 8 more
Caused by: java.lang.ClassNotFoundException: org.bouncycastle.operator.OperatorCreationException cannot be found by org.openhab.core.io.jetty.certificate_2.5.0.201905172343
        at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:484) ~[?:?]
        at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:395) ~[?:?]
        at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:387) ~[?:?]
        at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:150) ~[?:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:?]
        at java.lang.Class.getDeclaredConstructors0(Native Method) ~[?:?]
        at java.lang.Class.privateGetDeclaredConstructors(Class.java:2671) ~[?:?]
        at java.lang.Class.getConstructor0(Class.java:3075) ~[?:?]
        at java.lang.Class.newInstance(Class.java:412) ~[?:?]
        at org.eclipse.osgi.internal.framework.BundleContextImpl.loadBundleActivator(BundleContextImpl.java:763) ~[?:?]
        at org.eclipse.osgi.internal.framework.BundleContextImpl.start(BundleContextImpl.java:716) ~[?:?]
        at org.eclipse.osgi.internal.framework.EquinoxBundle.startWorker0(EquinoxBundle.java:933) ~[?:?]
        at org.eclipse.osgi.internal.framework.EquinoxBundle$EquinoxModule.startWorker(EquinoxBundle.java:309) ~[?:?]
        at org.eclipse.osgi.container.Module.doStart(Module.java:581) ~[?:?]
        ... 8 more
```
This is one of the last entries after starting...
```
2019-05-18 20:23:13.931 [ERROR] [org.apache.karaf.features.internal.service.BootFeaturesInstaller] - Error installing boot features
org.apache.karaf.features.internal.util.MultiException: Error restarting bundles:
        Error starting module.
        at org.apache.karaf.features.internal.service.Deployer.deploy(Deployer.java:1005) ~[?:?]
        at org.apache.karaf.features.internal.service.FeaturesServiceImpl.doProvision(FeaturesServiceImpl.java:1058) ~[?:?]
        at org.apache.karaf.features.internal.service.FeaturesServiceImpl.lambda$doProvisionInThread$13(FeaturesServiceImpl.java:994) ~[?:?]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:?]
        at java.lang.Thread.run(Thread.java:748) [?:?]
        Suppressed: org.osgi.framework.BundleException: Error starting module.
                at org.eclipse.osgi.container.Module.doStart(Module.java:590) ~[?:?]
                at org.eclipse.osgi.container.Module.start(Module.java:449) ~[?:?]
                at org.eclipse.osgi.internal.framework.EquinoxBundle.start(EquinoxBundle.java:383) ~[?:?]
                at org.eclipse.osgi.internal.framework.EquinoxBundle.start(EquinoxBundle.java:402) ~[?:?]
                at org.apache.karaf.features.internal.service.BundleInstallSupportImpl.startBundle(BundleInstallSupportImpl.java:161) ~[?:?]
                at org.apache.karaf.features.internal.service.FeaturesServiceImpl.startBundle(FeaturesServiceImpl.java:1149) ~[?:?]
                at org.apache.karaf.features.internal.service.Deployer.deploy(Deployer.java:997) ~[?:?]
                at org.apache.karaf.features.internal.service.FeaturesServiceImpl.doProvision(FeaturesServiceImpl.java:1058) ~[?:?]
                at org.apache.karaf.features.internal.service.FeaturesServiceImpl.lambda$doProvisionInThread$13(FeaturesServiceImpl.java:994) ~[?:?]
                at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:?]
                at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:?]
                at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:?]
                at java.lang.Thread.run(Thread.java:748) [?:?]
        Caused by: java.lang.NoClassDefFoundError: org/bouncycastle/operator/OperatorCreationException
                at java.lang.Class.getDeclaredConstructors0(Native Method) ~[?:?]
                at java.lang.Class.privateGetDeclaredConstructors(Class.java:2671) ~[?:?]
                at java.lang.Class.getConstructor0(Class.java:3075) ~[?:?]
                at java.lang.Class.newInstance(Class.java:412) ~[?:?]
                at org.eclipse.osgi.internal.framework.BundleContextImpl.loadBundleActivator(BundleContextImpl.java:763) ~[?:?]
                at org.eclipse.osgi.internal.framework.BundleContextImpl.start(BundleContextImpl.java:716) ~[?:?]
                at org.eclipse.osgi.internal.framework.EquinoxBundle.startWorker0(EquinoxBundle.java:933) ~[?:?]
                at org.eclipse.osgi.internal.framework.EquinoxBundle$EquinoxModule.startWorker(EquinoxBundle.java:309) ~[?:?]
                at org.eclipse.osgi.container.Module.doStart(Module.java:581) ~[?:?]
                ... 12 more
        Caused by: java.lang.ClassNotFoundException: org.bouncycastle.operator.OperatorCreationException cannot be found by org.openhab.core.io.jetty.certificate_2.5.0.201905172343
                at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:484) ~[?:?]
                at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:395) ~[?:?]
                at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:387) ~[?:?]
                at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:150) ~[?:?]
                at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:?]
                at java.lang.Class.getDeclaredConstructors0(Native Method) ~[?:?]
                at java.lang.Class.privateGetDeclaredConstructors(Class.java:2671) ~[?:?]
                at java.lang.Class.getConstructor0(Class.java:3075) ~[?:?]
                at java.lang.Class.newInstance(Class.java:412) ~[?:?]
                at org.eclipse.osgi.internal.framework.BundleContextImpl.loadBundleActivator(BundleContextImpl.java:763) ~[?:?]
                at org.eclipse.osgi.internal.framework.BundleContextImpl.start(BundleContextImpl.java:716) ~[?:?]
                at org.eclipse.osgi.internal.framework.EquinoxBundle.startWorker0(EquinoxBundle.java:933) ~[?:?]
                at org.eclipse.osgi.internal.framework.EquinoxBundle$EquinoxModule.startWorker(EquinoxBundle.java:309) ~[?:?]
                at org.eclipse.osgi.container.Module.doStart(Module.java:581) ~[?:?]
                ... 12 more
```

--- a/bundles/org.openhab.core.io.jetty.certificate/pom.xml
+++ b/bundles/org.openhab.core.io.jetty.certificate/pom.xml
@@ -17,7 +17,6 @@
       <groupId>org.bouncycastle</groupId>
       <artifactId>bcpkix-jdk15on</artifactId>
       <version>1.52</version>
-      <scope>provided</scope>
     </dependency>
   </dependencies>
 

